.github/initrd/alma.conf

```diff
@@ -0,0 +1,14 @@
+[Distribution]
+Distribution=alma
```

Comment:
Alma, Rocky and CentOS are pretty much the same, so let's only provide a config file for centos and not alma/rocky